In [1]:
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt
import matplotlib.ticker as ticker
from IPython.core.display import display, HTML
import os
%matplotlib inline

## Question 27
Which is cheaper: eating out or dining in?  The mean cost of a flank steak, broccoli, and rice bought at the grocery store is \$13.04 (Money.msn website, November 7, 2012).  A sample of 100 neighborhood restaurants showed a mean price of \$12.75 and a standard deviation of $2 for a comparable restaurant meal.

A. Develop appropriate hypotheses for a test to determine whether the sample data support the conclusion that the mean cost of a restaurant meal is less than fixing a comparable meal at home.

B. Using the sample from the 100 restaurants, what is the p – value?

C. At α = .05, what is your conclusion?

D.  Repeat the preceding hypothesis test using the critical value approach.

This question is asking if the cost difference of easting out is statistically significant. More specifically, it's asking specifically whether eating out is less than eating in, so they are asking a 1-tailed question.

The general equation you'll use for this problem is 
$$Z = \frac{\bar{x}-\mu_o}{\frac{s}{\sqrt{n}}}$$

where 
* $s$ is the standard deviation in restaurant price ($\$2$)
* $\mu_o$ is the null hypothesis mean (in this case, it's the grocery store price $\$13.04$)
* $n$ is the number of samples (100 restaurants)
* $\bar{x}$ is the sample mean (here, it's the $\$12.75$ mean price of a restaurant meal)
* $\mu$ is the actual mean price of all restaurants. We don't know this value, but we have the mean of a sample of 100 restaurants.
* $Z$ is the critical value

The null hypothesis ($H_o$) is that there isn't a statistically significant difference in price between restaurants and eating out ($\mu = \mu_o$)


$$H_o: \mu = \mu_o$$
$$H_a: \mu < \mu_o$$



In [2]:
z = (12.75 - 13.04)/(2/(100**0.5))
print(z)

-1.4499999999999957


That's about -1.45. Looking at a [standard normal table](https://en.wikipedia.org/wiki/Standard_normal_table) at the value for -1.45 (same as for 1.45, as the normal distribution is symmetrical), we see that the test statistic ($f(Z)$) at that level is 0.43943+0.5 = .93943, and $p-value = 1 - f(Z)$, so the $p-value = 0.061$. That's above the $\alpha = 0.05$ significance level, so we cannot reject the null hypothesis that eating out is significantly cheaper. 

0.06057000000000001